ols/app/endpoints/ols.py

```diff
@@ -163,7 +165,7 @@ def conversation_request(
 def process_request(
     auth: Any, llm_request: LLMRequest
 ) -> tuple[
-    str, str, str, list[CacheEntry], list[Attachment], bool, dict[str, float], str
+    str, str, str, list[CacheEntry], list[Attachment], bool, dict[str, float], str, str
```

Comment:
hmm sooner or later it would be valuable to create a model class for this tuple

Reply:
or split